values.yaml

```diff
@@ -78,6 +78,7 @@ jenkins:
         PRIVATE_DOCKER_REGISTRY: docker-private.packages.nuxeo.com
         PUBLIC_DOCKER_REGISTRY: docker.packages.nuxeo.com
         YUM_REGISTRY: http://nexus.platform/repository/yum-registry/
+        DOCKER_REGISTRY: docker.$NAMESPACE.dev.nuxeo.com
```

Comment:
Can you re-order to respect alphanumeric?

Reply:
will be taken to account